dapps/satoshi-multiplayer/contract/sources/tournament.move

```diff
@@ -17,6 +17,16 @@ module contract::tournament {
     use sui::dynamic_object_field as dof;
 
     // Structs.
+
+    // admin capability
+    // Gives person havining it the ability to end rounds.
+    // This is in case a player does not act on time.
+    // Will be removed in the future.
+    struct AdminCap has key {
+        id: UID,
+        admin_address: address,
```

Comment:
Update your comments to start with a capital letter and rename `admin_address` to `address`.

Reply:
Renaming `admin_address` to `address` is not possible, `address` is used for the address type. I can rename it to `admin` if we don't like `admin_address`.

Reply:
Ok, keep admin_address as it is then.